nextcord/webhook/sync.py

```diff
@@ -1019,6 +1025,7 @@ def edit_message(
         files: List[File] = MISSING,
         attachments: List[Attachment] = MISSING,
         allowed_mentions: Optional[AllowedMentions] = None,
+        thread: Snowflake = MISSING,
```

Comment:
Here it is named `thread`, and in other methods it is `thread_id` ? I think it is better to stick to one name.

Reply:
No, it's not a thread object.

Reply:
Aaah, I see what you mean. Why is it not just accepting `Thread` here?

Reply:
> Here it is named `thread`, and in other methods it is `thread_id` ? I think it is better to stick to one name.

I'll have to double check this but I think I just kept the same inconsistency that sending in threads has. I thought it was weird too

Reply:
> In this thread is a thread object, and in the other method thread_id is a thread id (which is an int ot a str, not sure) so the usage of var names is correct

thread_id is an int, btw

Reply:
> So you just need to pass sth with an .id attribute (which is constructible using nextcord.Object(id=1234) ) instead of fetching the full thread

So why not just use the actual id? xD

Reply:
Well, but I meant not to take the object itself. Rename it thread_id give typehint as int, because we don't need the WHOLE object, just its id.

Reply:
Should I do that? If so, send() should also be changed for consistency

Reply:
> Well, but I meant not to take the object itself. Rename it thread_id (or smth) give typehint as int, because we don't need the WHOLE object, just its id.

I don't like that approach. From the end user perspective, it seems more natural to just provide the thread object if you have one. And if you don't have one, then we have the `Object` placeholder to help out so developers don't have to write for unions of objects and ints.

Reply:
This pattern of differing items based on the area they are present makes sense when looking at the library as it currently stands